submissions/OgeEjim/html-forms/productpageforcinnamoncakes.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
You should have done product page task before contact form task. Is there a reason why you are redoing it again? Also, the product page shouldn't even be inside `html-forms` folder

Reply:
oh no, I am not redoing it, I added it to the folder so I could link both pages the way task 148 required. should I take it out?

Reply:
> You should have done product page task before contact form task. Is there a reason why you are redoing it again? Also, the product page shouldn't even be inside `html-forms` folder it ought to be in your `HTML` folder

Ma, I have removed the productpage and put the images in a folder